README.md

```diff
@@ -77,24 +87,6 @@ docker run -p 8080:80 -v ${PWD}:/spec docker.io/redocly/cli join spec/docs.yaml
 docker run -p 8080:80 -v ${PWD}:/spec docker.io/redocly/cli build-docs spec/index.yaml --output=docs/index.html --config=spec/redocly.yaml
 ```
 
-## Connecting from an Engine
```

Comment:
This part is covered in "Quick Start Guide" section. We don't need them any more.